clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -30,6 +30,15 @@ set(LLVM_LINK_COMPONENTS
   TransformUtils
   )
 
+# Workaround for MSVC ARM64 performance regression: disable all optimizations (/Od)
```

Comment:
I think this comment also should link to the issue on the Microsoft side so that we can know to remove this later.

Reply:
I was not sure if the link is acceptable to mention there. Will do, thanks!